bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/PCMConverter.java

```diff
@@ -155,139 +179,171 @@ public static String computeCompleteName(AbstractPCMVertex<?> vertex) {
                 .getEntityName();
     }
 
-    private DataFlowDiagramAndDictionary processPalladio(FlowGraphCollection flowGraphCollection) {
+    @SuppressWarnings("unchecked")
```

Comment:
Please address the issue rather than suppressing the warning

Reply:
Fixed in 172aeab80ebf